sdk/textanalytics/Azure.AI.TextAnalytics/src/TextSentimentScores.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// The sentiment confidence scores, by sentiment class label.
+    /// </summary>
+    public class TextSentimentScores
```

Comment:
Is `SentimentConfidenceScorePerLabel` not a good name here? Forgive me if I'm confusing the types again.

Reply:
the property the user calls is `SentimentScores`, so for me, `TextSentimentScores` sounds good.
`SentimentConfidenceScorePerLabel ` is super explicit on what it is, which I like, but seems too long.
Will bring up for discussion with other languages

Reply:
Could shorten it to `SentimentScorePerLabel`.